shade_janitor/resources.py

```diff
@@ -54,43 +48,36 @@ def _add_floatingip(self, fip):
                         'floating_ip': fip.floating_ip_address,
                         })
 
-    def check_instance_blacklisted(self, instance):
-        skip_it = False
-        for entry in ['jenkins', 'slave', 'mirror']:
-            if entry in instance.name:
-                skip_it = True
-                break
-        return skip_it
+    def is_blacklisted(self, instance):
+        return instance.name in self.BLACKLIST
```

Comment:
This logic also checks for blacklisted instances
self.BLACKLIST is ['jenkins', 'slave', 'mirror']

Also, there is no substring in removed code/logic